sharepoint/sharepoint-ps/sharepoint-online/Start-SPOTenantRename

```diff
@@ -0,0 +1,132 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://docs.microsoft.com/powershell/module/sharepoint-online/start-spotenantrename
+applicable: SharePoint Online
+title: Start-SPOTenantRename
+schema: 2.0.0
+author: WayneEwington
+ms.author: waynewin
+ms.reviewer:
+---
+
+# Start-SPOTenantRename
+
+## SYNOPSIS
+
+ [!IMPORTANT]
+> - This feature is available only for customers who have been accepted into the Private Preview program. [Request to join the Private Preview](https://aka.ms/SPOTenantRenameNomination). Note that submitting a nomination does not guarantee acceptance into the program. We review the nominations on a case-by-case basis and will contact you if you're accepted.
+> - This feature is not available for organizations that have set up multi-geo.
+
+Starts a job to change the SharePoint domain name for your orgnization in Microsoft 365. For example, if the name of your organization changes from "Contoso" to "Fabrikam," you can change contoso.sharepoint.com to fabrikam.sharepoint.com.
+
+> [!WARNING]
+> Changing your SharePoint domain name might take several hours to days depending on the number of sites and OneDrive users that you have. We strongly recommend that you make this change during a period of low usage (like a weekend) and tell users to avoid accessing SharePoint and OneDrive content during the change. In addition, any actions that create new OneDrives and sites (such as creating a new team or private channel in Microsoft Teams) will be temporarily blocked during the rename. 
+
+## SYNTAX
+
+```Powershell
+Start-SPOTenantRename -DomainName <string> -ScheduledDateTime <datetime> [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+This command schedules a rename of the SharePoint domain name for your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+```powershell
+Start-SPOTenantRename -DomainName "fabrikam" -ScheduledDateTime "2021-12-31T10:25:00"
+```
+
+Starts the rename of the SharePoint domain name to fabrikam that is scheduled for 31st December 2021 at 10:25:00am.
+
+## PARAMETERS
+
+### -DomainName
+
+Specifies the domain name that the current SharePoint domain name will be renamed to. This is the part before "sharepoint.com" or "onmicrosoft.com".
+
+> [!NOTE]
+> The domain name must already have been sucessfully added to Azure AD as per the instructions at [Step 1:Add the new domain name](https://aka.ms/SPOTenantRename#step-1-add-the-new-domain-name). If the domain name does not exist or was not sucessfully added, then this cmdlet will return an error.
+
+```yaml
+Type: String
+Parameter Sets: Default
```

Comment:
If there is only one parameter set and it is not named, this probably is just `Parameter Sets: (All)`